commands/active.go

```diff
@@ -12,22 +12,8 @@ func cmdActive(c *cli.Context) {
 		log.Fatal("Error: Too many arguments given.")
 	}
 
-	certInfo := getCertPathInfo(c)
```

Comment:
This pattern is / was all over the place, and it's super repetitive, so it's been replaced by a call to `getStore` which is a helper function that performs this logic.